src/getInitialMetadata.js

```diff
@@ -1,32 +1,17 @@
-const {
-  APP_NAME,
-  APP_VERSION,
-  DOCTYPE_CONTACTS_VERSION,
-  ADD_COZY_METADATA
-} = require('./constants')
+const { APP_NAME } = require('./constants')
 
 module.exports = (etag, resourceName, contactAccountId) => {
   let additionalData = {}
-  if (ADD_COZY_METADATA) {
-    const now = new Date().toISOString()
-    additionalData = {
-      cozyMetadata: {
-        doctypeVersion: DOCTYPE_CONTACTS_VERSION,
-        createdAt: now,
-        createdByApp: APP_NAME,
-        createdByAppVersion: APP_VERSION,
-        importedFrom: APP_NAME,
-        importedAt: now,
-        updatedAt: now,
-        updatedByApps: [APP_NAME],
-        sync: {
-          [contactAccountId]: {
-            konnector: APP_NAME,
-            lastSync: now,
-            remoteRev: etag,
-            id: resourceName,
-            contactsAccountsId: contactAccountId
-          }
+  const now = new Date().toISOString()
+  additionalData = {
```

Comment:
I think we could just return the new object, we don't need the `additionalData` variable anymore.
And since we're only adding the custom cozy metadata here, I'm not sure we need this function at all, I think this could be inline in the main script ?